### PR TITLE
refactor: renames notifications to notifier

### PR DESCRIPTION
**Pull request by jurajpiar**
- renames notifications to notifier as per https://rsklabs.atlassian.net/browse/RMKT-692
- turns inner repos into submodules

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[submodule "notifier-dev"]
+	path = notifier-dev
+	url = git@github.com:rsksmart/rif-marketplace-notifier.git
+[submodule "rns-dev"]
+	path = rns-dev
+	url = git@github.com:rsksmart/rif-marketplace-rns-dev-env.git
```
Comment (0xartem) [inline .gitmodules]: why don't we just use the nfts repo?

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+[submodule "storage-dev"]
+	path = storage-dev
+	url = git@github.com:rsksmart/rif-marketplace-dev.git
```
Comment (0xartem) [inline .gitmodules]: does it link itself?

File: storage-dev/migrations/2_white_listing.js [hunk 3]
```diff
@@ -1,106 +0,0 @@
-const fs = require("fs");
```
Comment (0xartem) [inline storage-dev/migrations/2_white_listing.js]: those files don't exist in the original SC repo, so we can't delete them.

**Review by 0xartem — DISMISSED** [review]
if we make the submodules change - we need to sync the dev subfolders with the original contract repos. Or revert it alltogether.

File: .gitmodules (hunk 1 above)
Comment (jurajpiar) [reply .gitmodules]: ~Wondered about that myself... I only used the repos that were already here (https://github.com/rsksmart/rif-marketplace-dev/pull/128/files#diff-1648d38ffb3ce5cb8b3cdd7e3938bcf10feebdf53f9200fe68d8181f9e795849L14)~

File: .gitmodules (hunk 2 above)
Comment (jurajpiar) [reply .gitmodules]: ~oh that's weird.... ok yea It seems I can't just use what's in the folder's package.json... I'll change this to point to the storage SC repo~

File: storage-dev/migrations/2_white_listing.js (hunk 3 above)
Comment (jurajpiar) [reply storage-dev/migrations/2_white_listing.js]: Aaaah I see... well spotted, I'll revert this commit then.

**Review by 0xartem — APPROVED** [review]
lgtm

**Pull request merged**